src/System.Private.Xml/src/System/Xml/Schema/UntypedStringTypeName.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Xml.Schema
+{
+    internal class UntypedStringTypeName
```

Comment:
??

Reply:
No idea what happened here.  I did most of this with VS refactorings, and I must have fat-fingered something and caused this to be added.  Will remove.